elixir/apps/web/lib/web/live/resources/edit.ex

```diff
@@ -50,6 +51,7 @@ defmodule Web.Resources.Edit do
             <.filters_form form={@form[:filters]} />
 
             <.connections_form
+              :if={is_nil(@params["site_id"])}
```

Comment:
@bmanifold those changes are somewhat similar to what you trying to do with "traffic filter" feature flag.